run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.aerosol

```diff
@@ -1384,49 +1384,92 @@ Warnings:                    1
-#0 HTAP_pFe_AIR1 -                                                                   -       -                  - -  -       pFe  11/29/66     1 4
-#0 HTAP_SO2_AIR2 $ROOT/HTAP/v2015-03/SO2/EDGAR_HTAP_SO2_AIR_CDS.generic.01x01.nc     emi_so2 2008-2010/1/1/0    C xy kg/m2/s SO2  11/29        1 4
-#0 HTAP_SO4_AIR2 -                                                                   -       -                  - -  -       SO4  11/29/63     1 4
-#0 HTAP_pFe_AIR2 -                                                                   -       -                  - -  -       pFe  11/29/66     1 4
-#0 HTAP_SO2_AIR3 $ROOT/HTAP/v2015-03/SO2/EDGAR_HTAP_SO2_AIR_CRS.generic.01x01.nc     emi_so2 2008-2010/1/1/0    C xy kg/m2/s SO2  11/29        1 4
-#0 HTAP_SO4_AIR3 -                                                                   -       -                  - -  -       SO4  11/29/63     1 4
-#0 HTAP_pFe_AIR3 -                                                                   -       -                  - -  -       pFe  11/29/66     1 4
-0 HTAP_NH3_IND   $ROOT/HTAP/v2015-03/NH3/EDGAR_HTAP_NH3_INDUSTRY.generic.01x01.nc    emi_nh3 2008-2010/1-12/1/0 C xy kg/m2/s NH3  -            1 4
-0 HTAP_NH3_POW   $ROOT/HTAP/v2015-03/NH3/EDGAR_HTAP_NH3_ENERGY.generic.01x01.nc      emi_nh3 2008-2010/1-12/1/0 C xy kg/m2/s NH3  -            1 4
-0 HTAP_NH3_RES   $ROOT/HTAP/v2015-03/NH3/EDGAR_HTAP_NH3_RESIDENTIAL.generic.01x01.nc emi_nh3 2008-2010/1-12/1/0 C xy kg/m2/s NH3  -            1 4
-0 HTAP_NH3_TRA   $ROOT/HTAP/v2015-03/NH3/EDGAR_HTAP_NH3_TRANSPORT.generic.01x01.nc   emi_nh3 2008-2010/1-12/1/0 C xy kg/m2/s NH3  -            1 4
-0 HTAP_NH3_AGR   $ROOT/HTAP/v2015-03/NH3/EDGAR_HTAP_NH3_AGRICULTURE.generic.01x01.nc emi_nh3 2008-2010/1-12/1/0 C xy kg/m2/s NH3  -            1 4
-)))HTAP
+# ==> HTAPv3 ship emissions are listed in the ship emissions section below
+#==============================================================================
+(((HTAPv3
+0 HTAPv3_CO_AGR   $ROOT/HTAPv3/v2022-12/$YYYY/HTAPv3_CO_0.1x0.1_$YYYY.nc   CO_AGR  2000-2018/1-12/1/0 C xy kg/m2/s SOAP  26/280   1 4
+0 HTAPv3_CO_ENE   $ROOT/HTAPv3/v2022-12/$YYYY/HTAPv3_CO_0.1x0.1_$YYYY.nc   CO_ENE  2000-2018/1-12/1/0 C xy kg/m2/s SOAP  26/280   1 4
+0 HTAPv3_CO_IND   $ROOT/HTAPv3/v2022-12/$YYYY/HTAPv3_CO_0.1x0.1_$YYYY.nc   CO_IND  2000-2018/1-12/1/0 C xy kg/m2/s SOAP  26/280   1 4
+0 HTAPv3_CO_TRA   $ROOT/HTAPv3/v2022-12/$YYYY/HTAPv3_CO_0.1x0.1_$YYYY.nc   CO_TRA  2000-2018/1-12/1/0 C xy kg/m2/s SOAP  26/280   1 4
+0 HTAPv3_CO_RCO   $ROOT/HTAPv3/v2022-12/$YYYY/HTAPv3_CO_0.1x0.1_$YYYY.nc   CO_RCO  2000-2018/1-12/1/0 C xy kg/m2/s SOAP  26/280   1 4
+0 HTAPv3_CO_SLV   $ROOT/HTAPv3/v2022-12/$YYYY/HTAPv3_CO_0.1x0.1_$YYYY.nc   CO_SLV  2000-2018/1-12/1/0 C xy kg/m2/s SOAP  26/280   1 4
+0 HTAPv3_CO_WST   $ROOT/HTAPv3/v2022-12/$YYYY/HTAPv3_CO_0.1x0.1_$YYYY.nc   CO_WST  2000-2018/1-12/1/0 C xy kg/m2/s SOAP  26/280   1 4
```

Comment:
These fields should be `HTAPv3_SOAP_` instead of `HTAPv3_CO_`.

Reply:
Now fixed in https://github.com/geoschem/geos-chem/commit/3f69125dbdf6ae180be14d943941683f4c7b9185

https://github.com/geoschem/geos-chem/blob/3f69125dbdf6ae180be14d943941683f4c7b9185/run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.aerosol#L1394-L1401